source/_integrations/fujitsu_fglair.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: Fujitsu FGLair
+description: Control your Fujitsu heat pump or air conditioner that uses the FGLair app
+ha_category:
+  - Climate
+ha_release: 2024.9
+ha_domain: fujitsu_fglair
+ha_integration_type: integration
+ha_codeowners:
+  - '@crevetor'
+ha_config_flow: true
+ha_platforms:
+  - climate
+ha_iot_class: "Cloud Polling"
+---
+
+The {{ page.title }} {% term integration %} provides support for Fujitsu heat pumps and air conditioners that use the FGLair app.
+To find out which app to use for your heat pump, check [the Fujitsu FGLair FAQ](https://www.fujitsu-general.com/global/support/faq/airstage-mobile/0127.html).
+
+## Prerequisites
+
+First, set up your device in the FGLair app before using this integration.
+To configure this integration, you will need the credentials (login and password) used to connect to the FGLair application.
+
+{% include integrations/config_flow.md %}
```

Comment:
Is there a list of supported devices or models?

Reply:
I haven't been able to find one.  I have a list of supported WLAN adapter models. As far as I understand it you can use 2 different types of WLAN adapters on the same heat pump model which would mean that the model is supported by this integration if you use one WLAN adapter model and by another if you choose the other one.